grails-app/domain/org/pih/warehouse/requisition/RequisitionItem.groovy

```diff
@@ -194,6 +194,37 @@ class RequisitionItem implements Comparable<RequisitionItem>, Serializable {
         }
     }
 
+    RequisitionItemStatus getDisplayStatus() {
```

Comment:
@alannadolny An afterthought. Is the above `getStatus()` still used? Seems like this one is adding two additional ifs at the top. Could we remove the other one and leave only this?

Btw GJ with tests.

Reply:
(Or if we are afraid that we can break something here after the two new ifs we could just call the other method.)

Reply:
@awalkowiak done